orchagent/main.cpp

```diff
@@ -346,28 +360,13 @@ int main(int argc, char **argv)
             gMacAddress = MacAddress(optarg);
             break;
         case 'r':
-            if (!strcmp(optarg, "0"))
-            {
-                gSairedisRecord = false;
-                gSwssRecord = false;
-            }
-            else if (!strcmp(optarg, "1"))
-            {
-                gSwssRecord = false;
-            }
-            else if (!strcmp(optarg, "2"))
-            {
-                gSairedisRecord = false;
-            }
-            else if (!strcmp(optarg, "3"))
-            {
-                continue; /* default behavior */
-            }
-            else
-            {
-                usage();
-                exit(EXIT_FAILURE);
-            }
+          // Disable all recordings if atoi() fails i.e. returns 0 due to
+          // invalid command line argument.
+          record_type = atoi(optarg);
+          if (record_type < 0 || record_type > 7) {
```

Comment:
Regarding style, I tried running the following:
```bash
git diff --name-only origin/master..HEAD | egrep "h$|cpp$" | xargs -t -n1 clang-format --style=Microsoft -i
```

And the diff is pretty significant including for existing files:

```
 cfgmgr/buffermgrd.cpp                              |   56 +-
 cfgmgr/coppmgrd.cpp                                |    2 +-
 cfgmgr/intfmgrd.cpp                                |   23 +-
 cfgmgr/macsecmgrd.cpp                              |   19 +-
 cfgmgr/natmgrd.cpp                                 |  100 +-
 cfgmgr/nbrmgrd.cpp                                 |   18 +-
 cfgmgr/sflowmgrd.cpp                               |    8 +-
 cfgmgr/teammgrd.cpp                                |   10 +-
 cfgmgr/tunnelmgrd.cpp                              |   17 +-
 cfgmgr/vlanmgrd.cpp                                |   28 +-
 cfgmgr/vrfmgrd.cpp                                 |   20 +-
 cfgmgr/vxlanmgrd.cpp                               |   28 +-
 orchagent/acltable.h                               |   52 +-
 orchagent/copporch.cpp                             |  308 +-
 orchagent/copporch.h                               |   76 +-
 orchagent/main.cpp                                 |  185 +-
 orchagent/orch.cpp                                 |  348 +-
 orchagent/orch.h                                   |  167 +-
 orchagent/orchdaemon.cpp                           |  389 +-
 orchagent/orchdaemon.h                             |   60 +-
 orchagent/p4orch/acl_rule_manager.cpp              | 3627 ++++----
 orchagent/p4orch/acl_rule_manager.h                |  282 +-
 orchagent/p4orch/acl_table_manager.cpp             | 1642 ++--
 orchagent/p4orch/acl_table_manager.h               |  175 +-
 orchagent/p4orch/acl_util.cpp                      | 1525 ++--
 orchagent/p4orch/acl_util.h                        |  429 +-
 orchagent/p4orch/mirror_session_manager.cpp        | 1266 +--
 orchagent/p4orch/mirror_session_manager.h          |  206 +-
 orchagent/p4orch/neighbor_manager.cpp              |  622 +-
 orchagent/p4orch/neighbor_manager.h                |   89 +-
 orchagent/p4orch/next_hop_manager.cpp              |  568 +-
 orchagent/p4orch/next_hop_manager.h                |  114 +-
 orchagent/p4orch/object_manager_interface.h        |   15 +-
 orchagent/p4orch/p4oidmapper.cpp                   |  283 +-
 orchagent/p4orch/p4oidmapper.h                     |  141 +-
 orchagent/p4orch/p4orch.cpp                        |  364 +-
 orchagent/p4orch/p4orch.h                          |   68 +-
 orchagent/p4orch/p4orch_util.cpp                   |  144 +-
 orchagent/p4orch/p4orch_util.h                     |  309 +-
 orchagent/p4orch/route_manager.cpp                 | 1008 +--
 orchagent/p4orch/route_manager.h                   |  154 +-
 orchagent/p4orch/router_interface_manager.cpp      |  672 +-
 orchagent/p4orch/router_interface_manager.h        |   95 +-
 orchagent/p4orch/tests/acl_manager_test.cpp        | 9157 +++++++++-----------
 orchagent/p4orch/tests/fake_consumerstatetable.cpp |   14 +-
 orchagent/p4orch/tests/fake_crmorch.cpp            |   72 +-
 orchagent/p4orch/tests/fake_dbconnector.cpp        |   50 +-
 .../p4orch/tests/fake_notificationconsumer.cpp     |   17 +-
 orchagent/p4orch/tests/fake_portorch.cpp           |  737 +-
 orchagent/p4orch/tests/fake_producertable.cpp      |   27 +-
 .../p4orch/tests/fake_subscriberstatetable.cpp     |   14 +-
 orchagent/p4orch/tests/fake_table.cpp              |  120 +-
 .../p4orch/tests/mirror_session_manager_test.cpp   | 1978 ++---
 orchagent/p4orch/tests/mock_response_publisher.h   |   27 +-
 orchagent/p4orch/tests/mock_sai_acl.cpp            |   86 +-
 orchagent/p4orch/tests/mock_sai_acl.h              |  146 +-
 orchagent/p4orch/tests/mock_sai_hostif.cpp         |   91 +-
 orchagent/p4orch/tests/mock_sai_hostif.h           |  133 +-
 orchagent/p4orch/tests/mock_sai_mirror.h           |   67 +-
 orchagent/p4orch/tests/mock_sai_neighbor.h         |   67 +-
 orchagent/p4orch/tests/mock_sai_next_hop.h         |   64 +-
 orchagent/p4orch/tests/mock_sai_next_hop_group.h   |   78 +-
 orchagent/p4orch/tests/mock_sai_policer.h          |   78 +-
 orchagent/p4orch/tests/mock_sai_route.h            |  193 +-
 orchagent/p4orch/tests/mock_sai_router_interface.h |   70 +-
 orchagent/p4orch/tests/mock_sai_serialize.cpp      |   12 +-
 orchagent/p4orch/tests/mock_sai_serialize.h        |   22 +-
 orchagent/p4orch/tests/mock_sai_switch.cpp         |   14 +-
 orchagent/p4orch/tests/mock_sai_switch.h           |   24 +-
 orchagent/p4orch/tests/mock_sai_udf.cpp            |   44 +-
 orchagent/p4orch/tests/mock_sai_udf.h              |   77 +-
 orchagent/p4orch/tests/mock_sai_virtual_router.h   |   69 +-
 orchagent/p4orch/tests/neighbor_manager_test.cpp   | 1476 ++--
 orchagent/p4orch/tests/next_hop_manager_test.cpp   | 1248 ++-
 orchagent/p4orch/tests/p4oidmapper_test.cpp        |  201 +-
 orchagent/p4orch/tests/p4orch_util_test.cpp        |  120 +-
 orchagent/p4orch/tests/return_code_test.cpp        |  272 +-
 orchagent/p4orch/tests/route_manager_test.cpp      | 3206 ++++---
 .../p4orch/tests/router_interface_manager_test.cpp | 1593 ++--
 orchagent/p4orch/tests/test_main.cpp               |  280 +-
 orchagent/p4orch/tests/wcmp_manager_test.cpp       | 3821 ++++----
 orchagent/p4orch/wcmp_manager.cpp                  | 1311 +--
 orchagent/p4orch/wcmp_manager.h                    |  280 +-
 orchagent/portsorch.h                              |  122 +-
 orchagent/request_parser.cpp                       |  225 +-
 orchagent/request_parser.h                         |   93 +-
 orchagent/response_publisher.cpp                   |  277 +-
 orchagent/response_publisher.h                     |   65 +-
 orchagent/response_publisher_interface.h           |   49 +-
 orchagent/return_code.h                            |  523 +-
 orchagent/saihelper.cpp                            |  301 +-
 orchagent/switchorch.cpp                           |  253 +-
 orchagent/switchorch.h                             |   61 +-
 orchagent/vrforch.cpp                              |   90 +-
 tests/mock_tests/aclorch_ut.cpp                    | 1835 ++--
 tests/mock_tests/fake_response_publisher.cpp       |   32 +-
 tests/mock_tests/mock_orchagent_main.h             |   28 +-
 97 files changed, 22405 insertions(+), 24617 deletions(-)
```

Reply:
1. Mostly we follow the same style in the file. For this PR, please address the new line comment and indentation for the added code. 
2. No. 
3. Not as part of this PR. 

In general, we try to catch it in review but if the changes are big, it could get missed. That's why there are discrepancies.

Reply:
Ok thank you